gpkit/solution_array.py

```diff
@@ -215,7 +223,7 @@ def results_table(data, title, minval=0, printunits=True, fixedcols=True,
                 lines.append(["", "", "", ""])
             if model is not "":
                 if not latex:
-                    lines.append([model+" | ", "", "", ""])
+                    lines.append([("modelname",), model, "", ""])
```

Comment:
I think the "modelname" bit deserves an inline comment, including mentioning that it gets specifically looked-for down below in the file.

This is somewhat of a hack, right?